test/util.py

```diff
@@ -1,20 +1,22 @@
 ''' Utilities for test. '''
-
 import os
+from typing import Dict, List
+from logging import getLogger, basicConfig, DEBUG
+
 import pyodbc
+from pyodbc import Connection
 
-from logging import getLogger, basicConfig, DEBUG
 
 basicConfig(level='DEBUG')
-logger = getLogger()
+logger: str = getLogger()
```

Comment:
Are you sure?

Reply:
:smile:

Reply:
@kabirbaidhya is the return type RootLogger? 
It returns the type when I print `logger`
```bash 
<class 'logging.RootLogger'>
```

Reply:
Yes.